conans/client/cache/cache.py

```diff
@@ -154,37 +154,38 @@ def config(self):
 
     @property
     def localdb(self):
-        return join(self.cache_folder, LOCALDB)
+        localdb_filename = os.path.join(self.cache_folder, LOCALDB)
+        return LocalDB.create(localdb_filename)
```

Comment:
Do we want to cache the created instance, just in case someone abuse calling ``cache.localdb``? Or convert it to a method instead of a property, something like ``create_localdb()``?

Reply:
I didn't want to make more changes. Before we were returning the path and creating the instance, now we are returning the instance just created. So, we are instantiating it the same number of times. Too many times.

I can create the localdb in the `__init__` method and then return the object that is already created `get_localdb()`. IMHO, if we want to cache at function level we should use a decorator to make it always in the same way.

Reply:
Agree